src/services/item/plugins/published/service.ts

```diff
@@ -58,12 +58,12 @@ export class ItemPublishedService {
 
     // item should be public first
     await itemTagRepository.getType(item, ItemTagType.Public, { shouldThrow: true });
-    const itemViews = await actionRepository.getForItem(item.path, {
+    const totalViews = await actionRepository.getForItem(item.path, {
```

Comment:
Maybe we should leverage the `getAggregationForItem` function, in order to perform the sum in sql and not in JS (performance concerns).